Get libxl to use the format "tapx.y" for emulated NICs, rather than "vifx.y-emu"

This is needed for backwards compat with qemu-dm, which seems to expect this
(for now at least).

diff --git a/tools/libxl/libxl.c b/tools/libxl/libxl.c
index 3f0a7ff..875151b 100644
--- a/tools/libxl/libxl.c
+++ b/tools/libxl/libxl.c
@@ -3561,7 +3561,7 @@ const char *libxl__device_nic_devname(libxl__gc *gc,
     case LIBXL_NIC_TYPE_VIF:
         return GCSPRINTF(NETBACK_NIC_NAME, domid, devid);
     case LIBXL_NIC_TYPE_VIF_IOEMU:
-        return GCSPRINTF(NETBACK_NIC_NAME TAP_DEVICE_SUFFIX, domid, devid);
+        return GCSPRINTF("tap%u.%d", domid, devid);
     default:
         abort();
     }
